include/ck_tile/ops/epilogue/cshuffle_epilogue.hpp

```diff
@@ -41,6 +41,12 @@ struct CShuffleEpilogueProblem
     static constexpr index_t kKPerXdl                      = kKPerXdl_;
     static constexpr index_t isCTransposed                 = isCTransposed_;
     static constexpr memory_operation_enum MemoryOperation = MemoryOperation_;
+    // NumMXdlPerWavePerShuffle used for in each shuffle iteration how many xdl tiles in M
+    // dimensions per wave
```

Comment:
Can you please use doxygen comment style?
```suggestion
    // NumMXdlPerWavePerShuffle - Define how many XDL tiles in M dimension are shuffled per single iteration per wave. 
```

Reply:
Done.